src/app/util/Url/Url.js

```diff
@@ -29,6 +29,7 @@ const getUrlParam = (match, location) => {
     const baseUrl = match.path;
     const currentUrl = location.pathname;
 
+    if (baseUrl === '/') return currentUrl.replace(baseUrl, '');
```

Comment:
What this one does?

Reply:
it is needed when we have a direct path like for e.g. scandipwa.local/super-category `currentUrl.replace(baseUrl, '')` , since here baseUrl = '/' it will return super-category. And if we will add substring on top of that first char will be removed.